package.json

```diff
@@ -1,17 +1,19 @@
 {
-  "name": "@goodbyteco/vivicReseachSite",
-  "version": "1.0.0",
-  "private": true,
-  "scripts": {
-    "dev": "astro dev",
-    "start": "astro dev",
-    "build": "astro build",
-    "preview": "astro preview"
-  },
-  "devDependencies": {
-    "@astrojs/renderer-vue": "^0.2",
-    "astro": "^0.22.20",
-    "autoprefixer": "^10.4.2",
-    "eslint-config-goodbyte-styleguide": "^0.0.7"
-  }
+	"name": "@goodbyteco/vivic",
+	"version": "1.0.0",
+	"private": true,
+	"scripts": {
+		"dev": "astro dev",
+		"build": "astro build",
+		"preview": "astro preview",
+		"lint": "eslint ."
+	},
+	"dependencies": {
+		"@astrojs/renderer-vue": "^0.4.0"
```

Comment:
I know this is sort of weird. But we're loading Vue client-side for the `Work.vue` component. I'm honestly not sure, but I would assume this is the dep that's providing the Vue that gets sent.

Reply:
(also bumped from `0.2.0` -> `0.4.0`)